### PR TITLE
Converted site to Jekyll

### PR DESCRIPTION
**Pull request by zenorocha**
As requested in #50.

### CODE REVIEW

File: _includes/basket-clear.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<h3 id="clear-doc">basket.clear</h3>
```
Comment (sindresorhus) [inline _includes/basket-clear.md]: Can you use Markdown headers?

Comment (zenorocha) [inline _includes/basket-clear.md]: I don't know how to use a markdown header with `id`.

**Comment by sindresorhus** [comment]
That was fast!

I don't think we need to split up the docs in so many files.

File: _includes/api.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+	<p>See the updated documentation below for more details.</p>
+</div>
+
+<ul>
+	<li><a href="#require-doc"><code>basket.require()</code></a></li>
+	<li><a href="#get-doc"><code>basket.get()</code></a></li>
+	<li><a href="#remove-doc"><code>basket.remove()</code></a></li>
+	<li><a href="#clear-doc"><code>basket.clear()</code></a></li>
+</ul>
```
Comment (sindresorhus) [inline _includes/api.html]: Markdown too?

**Comment by wibblymat** [comment]
There was already a MD version of the docs part in documentation.md if you want to copy/paste rather than rewriting that part by hand.

File: _includes/basket-clear.md (hunk 1 above)
Comment (sindresorhus) [inline _includes/basket-clear.md]: Id is autogenerated by GitHub based on the header title.

**Comment by zenorocha** [comment]
Oh man, I wish I knew that earlier, I rewrote each part by hand =/

File: _includes/api.html (hunk 2 above)
Comment (zenorocha) [inline _includes/api.html]: the `<ul>` is easy, but when I tried to convert the content inside of `div.boxout` it didn't render like markdown.

Comment (sindresorhus) [inline _includes/api.html]: No worries, just convert it to plain markdown, don't really need the style.

**Comment by zenorocha** [comment]
Done @sindresorhus :)

File: index.html [hunk 3]
```diff
@@ -1,272 +1,16 @@
...
+{% include project-history.md %}
+{% include compatibility.md %}
+{% include about-localstorage.md %}
+{% include api.md %}
+{% include basket-require.md %}
+{% include basket-get.md %}
+{% include basket-remove.md %}
+{% include basket-clear.md %}
+{% include the-future.md %}
+{% include team.md %}
```
Comment (sindresorhus) [inline index.html]: Still too many files. How about one file called `index.md` containing everything except the `api.md` which contains all the `basket_*` files.

Resulting files:

index.md
api.md

Comment (zenorocha) [inline index.html]: I prefer small files, but ok.

**Comment by zenorocha** [comment]
Is it ok for you now @sindresorhus?

**Comment by sindresorhus** [comment]
@zenorocha Landed. Thanks for doing this :D

**Comment by zenorocha** [comment]
Great xD

**Pull request closed without merging**